.rubocop.yml

```diff
@@ -26,6 +26,21 @@ Style/FrozenStringLiteralComment:
 Layout/EndOfLine:
   EnforcedStyle: lf
 
+Layout/ClassStructure:
+  Enabled: true
+  Categories:
+    include:
+      - prepend
```

Comment:
Why is just `prepend` here? Shouldn't `include` the here as well? And maybe `extend` as well. Generally I'd name this `module_inclusion` or something like this.

Reply:
Nice. I’m basically cheating here and making prepend become include. Let’s wrap all of them into module_inclusion then.